terraform-cloud-enterprise-private-module-registry-placeholder.tf

```diff
@@ -0,0 +1,37 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# THIS IS A PLACEHOLDER MODULE SO YOU CAN ADD THIS REPO TO THE PRIVATE MODULE REGISTRY IN TERRAFORM CLOUD / ENTERPRISE
+#
+# All the real modules in this repo are in the /modules sub-folder, but Terraform Cloud and Terraform Enterprise
+# require that you have Terraform code in the root of the repo, so this placeholder .tf file is necessary to allow you
+# to add the modules in this repo to your Private Module Registry. The real modules will show up in the Private
+# Module Registry UI under the "sub-modules" drop-down and you can use a sub-module named <MODULE_NAME> by adding
+# //modules/<MODULE_NAME> to the source URL:
+#
+# module "<NAME>" {
+#   source  = "<TERRAFORM_URL>/<ORG_NAME>/<REPO_NAME>/<CLOUD>//modules/<MODULE_NAME>"
+#   version = "<VERSION>"
+# }
+#
+# For example, to use version v0.12.4 of the vpc-app module from the terraform-aws-vpc repo with Terraform Cloud and an
+# organization named acme:
+#
+# module "vpc" {
+#   source  = "app.terraform.io/acme/vpc/aws//modules/vpc-app"
```

Comment:
NIT: random question, but I never asked until now - why do we need to use the `//` 👀

Reply:
It is so relative paths work. When you include a double slash like `<REPO>//modules/vpc-app`, Terragrunt/Terratest `git clone` ALL of `<REPO>`, and then run your command (e.g,. `apply`) inside of `modules/vpc-app`. Since the entire `<REPO>` is cloned, then any relative paths in `vpc-app` to other folders in `<REPO>` will work as you expect.